fix_subscription_end_date_simple.py

```
@@ -0,0 +1,100 @@
+        subscription = stripe.Subscription.retrieve(stripe_sub_id)
+
+        print(f"   Status: {subscription.status}")
+        print(f"   Cancel at period end: {subscription.cancel_at_period_end}")
+        print(f"   Current period end: {subscription.current_period_end}")
+
+        if subscription.current_period_end:
+            end_date = datetime.fromtimestamp(subscription.current_period_end)
+            print(f"   End date: {end_date.strftime('%B %d, %Y')}")
+
+            # Update database
+            print(f"\n[INFO] Updating database...")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)